### DISCUSSION TITLE
Docs: Auth class names not updated on website

### DISCUSSION THREAD

**Discussion by aawadaptix** [encode/httpx#3137] [Potential Issue] [answered]
Hi, apologies if this is a duplicate, I did not find an existing issue.

On the docs website https://www.python-httpx.org/advanced/authentication/ the class names on the Authentication page don't match the current names (`BasicAuthentication --> BasicAuth`). However I can see that in the repo's docs dir the source is up to date. So maybe an Actions issue?

**Comment by lovelydinosaur** [comment, marked as answer]
This is resolved in https://github.com/encode/httpx/pull/3112 ... as things currently stand it won't be updated until we roll a release.

**Reply by aawadaptix** [reply]
OK thanks!
